routing/transit_graph.cpp

```diff
@@ -176,7 +178,9 @@ Segment TransitGraph::GetNewTransitSegment() const
 }
 
 void TransitGraph::AddGate(transit::Gate const & gate, FakeEnding const & ending,
-                           map<transit::StopId, Junction> const & stopCoords, bool isEnter)
+                           map<transit::StopId, Junction> const & stopCoords, bool isEnter,
+                           map<transit::StopId, set<Segment>> & stopToBack,
```

Comment:
How about to assign a short and meaningful name to ```map<transit::StopId, set<Segment>>```?

Reply:
done